src/lib/components/Hoverable.svelte

```diff
@@ -10,6 +10,6 @@
   }
 </script>
 
-<div on:mouseenter="{enter}" on:mouseleave="{leave}">
+<div on:mouseenter={enter} on:mouseleave={leave} role="button" tabindex="0">
```

Comment:
not sure about this

Reply:
mouseleave handlers must have an ARIA role(a11y-no-static-element-interactions)
Elements with the 'button' interactive role must have a tabindex value.(a11y-interactive-supports-focus)
These are the errors, one for not having a role and another for the tabindex. It seems like it's now mandatory, at least with the button role.